.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "stage": 2
+    "stage": 2
```

Comment:
Looks like there's no standards for JSON style, since it's a serialization format. We can discuss if it's better with two or four. Personally I prefer two spaces because it's closer to javascript style.